### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
# Description
Updated EN -> JA documentation translations

- English texts remain in this PR, which will be translated next time. 

- Code comments (starts with #) are no longer translated. When changes are made to the code, code comments that were previously translated to Japanese may be reverted to English, but this is allowed.

### CODE REVIEW

**Review by devinmarieb — APPROVED** [review]

File: jekyll/_cci2_ja/authorize-google-cloud-sdk.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+3. 便宜上、次の 2 つの環境変数を CircleCI プロジェクトに追加します。
+    - `GOOGLE_PROJECT_ID`: GCP プロジェクトの ID
+    - `GOOGLE_COMPUTE_ZONE`: デフォルトの [Compute Engine のゾーン](https://cloud.google.com/compute/docs/regions-zones/)
+
+#### きめ細かい権限の追加
+{: #adding-granular-permissions }
+
+GCR へのコンテナ イメージのプッシュに問題が発生している場合は、デフォルトの `service account` によって提供されるものよりもきめ細かい権限が必要とされている可能性があります。 権限の変更は、Cloud Storage の [IAM コンソール](https://console.cloud.google.com/iam-admin/iam/project)で許可することができます。
+
+Refer to the Cloud Storage [permission documentation](https://cloud.google.com/storage/docs/access-control/iam-permissions) to learn more about Identity and Access Management (IAM) permissions.
```
Comment (ogii) [inline jekyll/_cci2_ja/authorize-google-cloud-sdk.md:57]: This sentence should be in Japanese.

Perhaps something like this?

```
Identity and Access Management (IAM) 権限の詳細については、Cloud Storage [permission documentation](https://cloud.google.com/storage/docs/access-control/iam-permissions) を参照してください。
```

File: jekyll/_cci2_ja/concepts.md [hunk 2]
```diff
@@ -641,7 +641,7 @@ Orb は、再利用可能なコードスニペットです。Orb を使用する
 
 {: #parallelism }
 
-プロジェクトに含まれるテストの数が多いほど、テストを 1 台のマシンで実行するのに時間がかかるようになります。 _並列実行_ により、指定した数の別々の Executor にテストを分散することができます。
+プロジェクトに含まれるテストの数が多いほど、テストを 1 台のマシンで実行するのに時間がかかるようになります。  _並列実行_ により、指定した数の別々の Executor にテストを分散することができます。
```
Comment (ogii) [inline jekyll/_cci2_ja/concepts.md:644]: Should there be spaces in the following text? `テストを 1 台`

Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/concepts.md:644]: Yes, we put a space before and after a number (although sometimes missing 😅 )

File: jekyll/_cci2_ja/authorize-google-cloud-sdk.md (hunk 1 above)
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/authorize-google-cloud-sdk.md:57]: Thank you! Applied.

File: jekyll/_cci2_ja/authorize-google-cloud-sdk.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+- CircleCI プロジェクト
+- GCP プロジェクト
+
+### Google Cloud SDK のインストール
+{: #installing-the-google-cloud-sdk }
+
+プライマリ コンテナでオペレーティング システムとして Debian を受け入れ可能な場合は、Google の基本 Docker イメージの使用を検討してください。 このイメージは、Docker Hub で [`google/cloud-sdk`](https://hub.docker.com/r/google/cloud-sdk/) として提供されています。
+
+それ以外の場合は、基本イメージのオペレーティング システムに対応する [Google Cloud SDK インストール手順](https://cloud.google.com/sdk/)に従ってください。
```
[the remark this answers is not in the document]
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/authorize-google-cloud-sdk.md:37]: Applied :)

File: jekyll/_cci2_ja/migrating-from-jenkins.md [hunk 4]
```diff
@@ -147,54 +147,54 @@ Below is a table of supported plugins that you can convert using the [CircleCI J
...
 {: #supported-syntax }
 
-現在サポートされているのは宣言型 (パイプライン)の `jenkinsfile` のみです。
+現在サポートされているのは宣言型 (パイプライン) の `Jenkinsfile` のみです。
 
 | Jenkinsfile 構文 | 近似する CircleCI 構文                                                                    | ステータス                                                                       |
 | -------------- | ----------------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
 | agent          | [Executor]({{site.baseurl}}/configuration-reference/#executors-requires-version-21) | 静的                                                                          |
-| post           | [when 属性]({{site.baseurl}}/configuration-reference/#the-when-attribute)             | [when]({{site.baseurl}}/configuration-reference/#the-when-attribute) を参照    |
+| post           | [when 属性]({{site.baseurl}}/configuration-reference/#the-when-attribute)             | [when]({{site.baseurl}}/ja/configuration-reference/#the-when-attribute) を参照 |
```
Comment (ogii) [inline jekyll/_cci2_ja/migrating-from-jenkins.md:197]:
```suggestion
| post           | [when 属性]({{site.baseurl}}/ja/configuration-reference/#the-when-attribute)             | [when]({{site.baseurl}}/ja/configuration-reference/#the-when-attribute) を参照 |
```

File: jekyll/_cci2_ja/migrating-from-jenkins.md [hunk 5]
```diff
@@ -231,39 +231,39 @@ The CircleCI [Jenkins Converter](https://circleci.com/developer/tools/jenkins-co
...
+[CircleCI ワークフロー]({{site.baseurl}}/ja/workflows/) (Jenkins のパイプラインに相当) が Jenkinsfile から `.circleci/config.yml` に転送されます。これにはブランチのフィルタリングも含まれます。 コンバーターは、意図しないビルドがトリガーされないように、[スケジュール化されたビルド]({{site.baseurl}}/ja/configuration-reference/#triggers)は転送しません。
 
 #### ジョブ
 {: #jobs }
 
-Many of the configuration options within CircleCI jobs do not have equivalents to Jenkins' offerings. CircleCI でのエクスペリエンスを高めるには、次の機能から始めることをお勧めします。
+CircleCI ジョブの多くの設定オプションには、相当する Jenkins の製品がありません。 CircleCI でのエクスペリエンスを高めるには、次の機能から始めることをお勧めします。
 
-- [Checkout code (コードのチェックアウト)]({{site.baseurl}}/configuration-reference/#checkout)
+- [コードのチェックアウト]({{site.baseurl}}/configuration-reference/#checkout)
```
Comment (ogii) [inline jekyll/_cci2_ja/migrating-from-jenkins.md:255]:
```suggestion
- [コードのチェックアウト]({{site.baseurl}}/ja/configuration-reference/#checkout)
```

File: jekyll/_cci2_ja/migrating-from-jenkins.md [hunk 6]
```diff
@@ -231,39 +231,39 @@ The CircleCI [Jenkins Converter](https://circleci.com/developer/tools/jenkins-co
...
 - [リソースクラス]({{site.baseurl}}/configuration-reference/#resource_class)
```
Comment (ogii) [inline jekyll/_cci2_ja/migrating-from-jenkins.md:256]:
```suggestion
- [リソースクラス]({{site.baseurl}}/ja/configuration-reference/#resource_class)
```

File: jekyll/_cci2_ja/migrating-from-jenkins.md [hunk 7]
```diff
@@ -231,39 +231,39 @@ The CircleCI [Jenkins Converter](https://circleci.com/developer/tools/jenkins-co
...
 - [並列実行]({{site.baseurl}}/configuration-reference/#parallelism)
```
Comment (ogii) [inline jekyll/_cci2_ja/migrating-from-jenkins.md:257]:
```suggestion
- [並列実行]({{site.baseurl}}/ja/configuration-reference/#parallelism)
```

File: jekyll/_cci2_ja/migrating-from-jenkins.md [hunk 8]
```diff
@@ -231,39 +231,39 @@ The CircleCI [Jenkins Converter](https://circleci.com/developer/tools/jenkins-co
...
-- キャッシュの[保存]({{site.baseurl}}/configuration-reference/#save_cache)と[リストア]({{site.baseurl}}/configuration-reference/#restore_cache)
+- キャッシュの[保存]({{site.baseurl}}/ja/configuration-reference/#save_cache)と[リストア]({{site.baseurl}}/ja/configuration-reference/#restore_cache)
 - [アーティファクトの保存]({{site.baseurl}}/configuration-reference/#store_artifacts)
```
Comment (ogii) [inline jekyll/_cci2_ja/migrating-from-jenkins.md:259]:
```suggestion
- [アーティファクトの保存]({{site.baseurl}}/ja/configuration-reference/#store_artifacts)
```

File: jekyll/_cci2_ja/runner-installation-mac.adoc [hunk 9]
```diff
@@ -39,6 +39,15 @@ logging:
   file: /Library/Logs/com.circleci.runner.log
 ```
 
+[#update-workdir-ownership]
+== Update the Working Directory Permission
+
+The CircleCI agent requires write permission to the directory containing the working directory. Change the ownership of that directory to `USERNAME`:
```
Comment (ogii) [inline jekyll/_cci2_ja/runner-installation-mac.adoc:45]: This area should be in Japanese

File: jekyll/_cci2_ja/runner-installation-mac.adoc [hunk 10]
```diff
@@ -95,23 +104,23 @@ sudo chmod 644 /Library/LaunchDaemons/com.circleci.runner.plist
         <key>RunAtLoad</key>
         <true/>
 
-        <!-- ランナーは実行状態を維持する必要があります -->
+        <!-- The agent needs to run at all times -->
```
Comment (ogii) [inline jekyll/_cci2_ja/runner-installation-mac.adoc:107]: should be in Japanese

File: jekyll/_cci2_ja/runner-installation-mac.adoc [hunk 11]
```diff
@@ -95,23 +104,23 @@ sudo chmod 644 /Library/LaunchDaemons/com.circleci.runner.plist
...
         <key>KeepAlive</key>
         <true/>
 
-        <!-- ランナーによるリソースの使用が macOS で制限されないようにする設定です-->
+        <!-- This prevents macOS from limiting the resource usage of the agent -->
```
Comment (ogii) [inline jekyll/_cci2_ja/runner-installation-mac.adoc:111]: should be in Japanese

File: jekyll/_cci2_ja/runner-installation-mac.adoc [hunk 12]
```diff
@@ -95,23 +104,23 @@ sudo chmod 644 /Library/LaunchDaemons/com.circleci.runner.plist
...
         <key>ProcessType</key>
         <string>Interactive</string>
 
-        <!-- 失敗時または更新後にランナーを再起動する頻度を増やします -->
+        <!-- Increase the frequency of restarting the agent on failure, or post-update -->
```
Comment (ogii) [inline jekyll/_cci2_ja/runner-installation-mac.adoc:115]: should be in Japanese

File: jekyll/_cci2_ja/runner-installation-mac.adoc [hunk 13]
```diff
@@ -95,23 +104,23 @@ sudo chmod 644 /Library/LaunchDaemons/com.circleci.runner.plist
...
         <key>ThrottleInterval</key>
         <integer>3</integer>
 
-        <!-- ランナーがシャットダウンするまで 10 分間待機します (ランナー自体はタスクが完了するまで待機します) -->
+        <!-- Wait for 10 minutes for the agent to shut down (the agent itself waits for tasks to complete) -->
```
Comment (ogii) [inline jekyll/_cci2_ja/runner-installation-mac.adoc:119]: should be in Japanese

File: jekyll/_cci2_ja/runner-installation-mac.adoc [hunk 14]
```diff
@@ -95,23 +104,23 @@ sudo chmod 644 /Library/LaunchDaemons/com.circleci.runner.plist
...
         <key>ExitTimeOut</key>
         <integer>600</integer>
 
-        <!-- ログの出力とローテーションの設定はランナー固有のものを使用します -->
+        <!-- The agent uses its own logging and rotation to file -->
```
Comment (ogii) [inline jekyll/_cci2_ja/runner-installation-mac.adoc:123]: should be in Japanese

File: jekyll/_cci2_ja/server-3-install-build-services.adoc [hunk 15]
```diff
@@ -720,7 +720,7 @@ gcloud iam service-accounts keys create circleci-server-vm-keyfile --iam-account
 
 . *サービスアカウントで Workload Identity を有効にします。*
 +
-この手順は、GKE で link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identity] を使用している場合のみ実行する必要があります。 Workload Identity を有効化する手順は、link:https://circleci.com/docs/server-3-install-prerequisites/#enabling-workload-identity-in-gke[こちら]を参照してください。
+この手順は、GKE で link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identity] を使用している場合のみ実行する必要があります。 Workload Identity を有効化する手順は、link:https://circleci.com/docs/server-3-install-prerequisites#enabling-workload-identity-in-gke[こちら]を参照してください。
```
Comment (ogii) [inline jekyll/_cci2_ja/server-3-install-build-services.adoc:723]:
```suggestion
この手順は、GKE で link:https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity[Workload Identity] を使用している場合のみ実行する必要があります。 Workload Identity を有効化する手順は、link:https://circleci.com/docs/ja/server-3-install-prerequisites#enabling-workload-identity-in-gke[こちら]を参照してください。
```

File: jekyll/_cci2_ja/server/operator/troubleshooting-and-support.adoc [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+[#generate-support-bundle]
+== サポートバンドルの生成
+
+サポートバンドルは、発生している問題を CircleCI エンジニアが診断、修正するためのものです。 通常、サポートチケットを作成する場合、サポートバンドルを提出するように求められます。
+
+サポート バンドルを生成するには、以下の手順に従います。
+
+=== 前提条件
+
+. まず、clageci-server がデプロイされており、kubectl によりクラスタ/名前空間にアクセスできることを確認します。
```
Comment (ogii) [inline jekyll/_cci2_ja/server/operator/troubleshooting-and-support.adoc:28]:
```suggestion
. まず、circleci-server がデプロイされており、kubectl によりクラスタ/名前空間にアクセスできることを確認します。
```

**Review by ogii — CHANGES_REQUESTED** [review]
I have made some comments and suggestions. Thank you!

**Comment by Mayuko-Sa** [comment]
Thank you @ogii I applied your suggestions! Please appove if it is okay 😃

**Review by ogii — APPROVED** [review]
Looks fine to me!

**Pull request merged**